mijia-homie/src/main.rs

```diff
@@ -460,21 +483,26 @@ async fn service_bluetooth_event_queue(
 async fn handle_bluetooth_event(
     state: Arc<Mutex<SensorState>>,
     event: MijiaEvent,
+    min_update_period: Duration,
```

Comment:
(optional) I'd be tempted to dump min_update_period on SensorState or something

Reply:
Done, though I'm not sure if it's better or worse.